### PR TITLE
Thief hotfix

### PR DESCRIPTION
**Pull request by TheShuEd**
## About the PR

fix some sloth review tasks
Small rebalancing of the contents of the kits
Disabling structure theft. So far, this type of task is not feasible. We need some new items for this in the form of separate PR
reduced the number of items in tasks with collections. The player has problems with free space

**Changelog**

:cl:
- tweak: The thief will temporarily not steal structures for the time being, as this task seems too difficult.
- add: Added Emag into thief's syndie kit
- add: Added 3 smoke grenades into thief's smuggler kit
- tweak: Replaced radio jammer to cyber pen into thief's communicator kit
- tweak: Replaced omnizine bottle to omega soap into thief's chemistry kit

### CODE REVIEW

**Comment by mirrorcult** [comment]
structure theft is extremely funny so i'd like to see it return in the future, but i get disabling it for now

**Comment by TheShuEd** [comment]
Yes, there are no normal ways to fulfill these goals yet. We need some way to carry them undetected, or maybe let the thief fly away on evac pods and consider the goal accomplished. 
Ideas:
An item that grants invisibility to another object or character (invisibility cloak). Can be thrown over a structure or worn on its own

Bluespace Glove. Works as a ripley.

**Comment by mirrorcult** [comment]
i think one potential solution would be the option to bring the structure to a predefined drop area, where it will count as 'stolen' and you simply need to hide it there. or being able to attach something to it that will mark it as 'stolen' but can be easily removed

**Comment by TheShuEd** [comment]
some "Fulton" thematic will be good

**Comment by deltanedas** [comment]
fultons that start linked to a thieves den :trollface:

File: Resources/Prototypes/Catalog/thief_toolbox_sets.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@
   - StorageImplanter
   - DnaScramblerImplanter
   - EphedrineChemistryBottle
-  - OmnizineChemistryBottle
+  - SoapOmega
```
Comment (mirrorcult) [inline Resources/Prototypes/Catalog/thief_toolbox_sets.yml:46]: update chemistry kit desc to say it has soap

**Comment by TheShuEd** [comment]
I'll also add a guidebook to this PR.

**Comment by TheShuEd** [comment]
nah, separate pr.
this seems more time consuming than I thought

File: Content.Server/GameTicking/Rules/Components/ThiefRuleComponent.cs [hunk 2]
```diff
@@ -41,9 +42,9 @@ public sealed partial class ThiefRuleComponent : Component
     public List<EntProtoId> StarterItems = new List<EntProtoId> { "ToolboxThief", "ClothingHandsChameleonThief" }; //TO DO - replace to chameleon thieving gloves whem merg
 
     /// <summary>
-    /// All Thiefes created by this rule
+    /// All Thieves created by this rule
     /// </summary>
-    public readonly List<EntityUid> ThiefMinds = new();
+    public readonly List<EntityUid> ThievesMinds = new();
```
Comment (metalgearsloth) [inline Content.Server/GameTicking/Rules/Components/ThiefRuleComponent.cs:47]: datafield

File: Content.Server/GameTicking/Rules/ThiefRuleSystem.cs [hunk 3]
```diff
@@ -53,23 +53,20 @@ private void OnPlayersSpawned(RulePlayerJobsAssignedEvent ev)
...
+                if (!GameTicker.IsGameRuleAdded(uid, gameRule))
                     continue;
 
-                thief.StartCandidates[player] = ev.Profiles[player.UserId];
+                foreach (var player in ev.Players)
+                {
+                    if (!ev.Profiles.ContainsKey(player.UserId))
+                        continue;
+
+                    thief.StartCandidates[player] = ev.Profiles[player.UserId];
```
Comment (metalgearsloth) [inline Content.Server/GameTicking/Rules/ThiefRuleSystem.cs:66]: TryGetValue here.

File: Content.Server/GameTicking/Rules/ThiefRuleSystem.cs [hunk 4]
```diff
@@ -125,8 +122,10 @@ public bool MakeThief(ICommonSession thief, bool AddPacified)
         });
 
         //Add Pacified
-        if (AddPacified)
+        if (addPacified) //This check is important because some servers may want to disable the thief's pacifism. Do not remove.
+        {
             EnsureComp<PacifiedComponent>(mind.OwnedEntity.Value);
+        }
```
Comment (metalgearsloth) [inline Content.Server/GameTicking/Rules/ThiefRuleSystem.cs:129]: Long-term this should just be using the antag code to add components.

File: Content.Server/GameTicking/Rules/ThiefRuleSystem.cs [hunk 5]
```diff
@@ -92,7 +89,7 @@ private void DoThiefStart(ThiefRuleComponent component)
         }
     }
 
-    public bool MakeThief(ICommonSession thief, bool AddPacified)
+    public bool MakeThief(ICommonSession thief, bool addPacified)
     {
         var thiefRule = EntityQuery<ThiefRuleComponent>().FirstOrDefault();
         if (thiefRule == null)
```
Comment (metalgearsloth) [inline Content.Server/GameTicking/Rules/ThiefRuleSystem.cs:95]: MakeThief still needs to tie the thief to the specified rule and not a random one.

**Review by metalgearsloth — DISMISSED** [review]

File: Content.Server/GameTicking/Rules/ThiefRuleSystem.cs (hunk 4 above)
Comment (TheShuEd) [reply Content.Server/GameTicking/Rules/ThiefRuleSystem.cs:129]: marked as To Do

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**